src/fileformats/ocd_georef_fields.cpp

```diff
@@ -0,0 +1,457 @@
+    { OcdGrid::Brazil, 19, 31980 }, // SIRGAS 2000 / UTM zone 20S
+    { OcdGrid::Brazil, 20, 31981 }, // SIRGAS 2000 / UTM zone 21S
+    { OcdGrid::Brazil, 21, 31982 }, // SIRGAS 2000 / UTM zone 22S
+    { OcdGrid::Brazil, 22, 31983 }, // SIRGAS 2000 / UTM zone 23S
+    { OcdGrid::Brazil, 23, 31984 }, // SIRGAS 2000 / UTM zone 24S
+    { OcdGrid::Brazil, 24, 31985 }, // SIRGAS 2000 / UTM zone 25S
+    { OcdGrid::Britain, 0, 27700 }, // OSGB 1936
+    { OcdGrid::Czechia, 1, 5514 }, // S-JTSK Krovak East North
+    { OcdGrid::Czechia, 2, 28403 }, // Pulkovo 1942 / Gauss-Kruger zone 3
+    { OcdGrid::Finland, 2, 2392 }, // KKJ / Finland zone 2
+    { OcdGrid::Finland, 5, 3067 }, // TM35FIN, ETRS89 / ETRS-TM35FIN
+    { OcdGrid::Finland, 12, 3879 }, // ETRS89 / GK25FIN
+    { OcdGrid::Finland, 25, 3132 }, // ETRS89 / ETRS-GK25FIN
+    { OcdGrid::Germany, 2, 31466 }, // DHDN, Gauss-Krüger Zone 2
+    { OcdGrid::Germany, 3, 31467 }, // DHDN, Gauss-Krüger Zone 3
+    { OcdGrid::Germany, 4, 31468 }, // DHDN, Gauss-Krüger Zone 4
+    { OcdGrid::Germany, 5, 31469 }, // DHDN, Gauss-Krüger Zone 5
+    { OcdGrid::GoogleTms, 0, 900913 }, // Google Mercator
+    { OcdGrid::Hungary, 0, 23700 }, // HD72 / EOV
+    { OcdGrid::Iceland, 4, 3057 }, // ISN93 / Lambert 1993
+    { OcdGrid::Israel, 0, 2039 }, // Israel 1993 / Israeli TM Grid
+    { OcdGrid::Slovakia, 1, 5514 }, // S-JTSK Krovak East North
+    { OcdGrid::Slovenia, 1, 3912 }, // MGI 1901 / Slovene National Grid
+    { OcdGrid::Sweden, 2, 3006 }, // SWEREF99 TM
+    { OcdGrid::Sweden, 3, 3007 }, // SWEREF99 12 00
+    { OcdGrid::Sweden, 4, 3008 }, // SWEREF99 13 30
+    { OcdGrid::Sweden, 5, 3009 }, // SWEREF99 15 00
+    { OcdGrid::Sweden, 6, 3010 }, // SWEREF99 16 30
+    { OcdGrid::Sweden, 7, 3011 }, // SWEREF99 18 00
+    { OcdGrid::Sweden, 8, 3012 }, // SWEREF99 14 15
+    { OcdGrid::Sweden, 9, 3013 }, // SWEREF99 15 45
+    { OcdGrid::Sweden, 10, 3014 }, // SWEREF99 17 15
+    { OcdGrid::Sweden, 11, 3015 }, // SWEREF99 18 45
+    { OcdGrid::Sweden, 12, 3016 }, // SWEREF99 20 15
+    { OcdGrid::Sweden, 13, 3017 }, // SWEREF99 21 45
+    { OcdGrid::Sweden, 14, 3018 }, // SWEREF99 23 15
+    { OcdGrid::Switzerland, 1, 21781 }, // CH1903 / LV03 -- Swiss CH1903 / LV03
+    { OcdGrid::Switzerland, 2, 2056 }, // CH1903+ / LV95 -- Swiss CH1903+ / LV95
+};
+
+/**
+ * MapperCrsRecord struct is used for passing CRS information around within the
+ * translation code.
+ */
+struct MapperCrsRecord
+{
+	/** CRS id, converted to a string via crs_string_map and passed to
+	 *  CRSTemplateRegistry::find() */
+	const MapperCrs crs_id;
+	/** CRS parameter, converted to a string and passed through to
+	 *  CRSTemplateRegistry::specValues() */
+	const int crs_param;
+};
+
+/**
+ * Plain translation from OCD grid/zone combination into a Mapper
+ * scheme.
+ *
+ * @see translateMapperToOcd
+ */
+MapperCrsRecord translateOcdToMapper(const int ocd_grid_id,
+                                     const int ocd_zone_id)
```

Comment:
Instead of a free function, I would prefer a "named constructor" i.e.  `MapperCrsRecord::fromOcdGridAndZone`. This is more expressive, for example with auto style:
~~~
auto rec = MapperCrsRecord::fromOcdGridAndZone(grid_id, zone_id);
~~~
...GridAndZone helps to get the arguments in the right order. However if it would take a single combined grid and zone  parameter, it might be just `MapperCrsRecord::fromOcd`. And...

Reply:
Renaming `MapperCrsRecord::translateOcdToMapper()` to `MapperCrsRecord::fromOcd()` would break symmetry. `translateOcdToMapper()` and `translateMapperToOcd()` form a pair. Either we rename both (to `fromOcd()` and `toOcd()`) or none.